src/NuGet.Indexing/AuxiliaryDataFileNames.cs

```diff
@@ -0,0 +1,14 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace NuGet.Indexing
+{
+    class AuxiliaryDataFileNames
```

Comment:
I do not think each auxiliary data file needs to be represented as an individual class. There aren't direct actionable items on each file as such. That would be an overkill of the design. 

That said, I have changed this file to be AuxiliaryFiles class which would hold the list of these files. If we were to add any new files in, we can just add it to the list in here and update the list of files update times accordingly from `NugetSeracherManager`.